app/reducers/index.js

```diff
@@ -1,9 +1,7 @@
-import {combineReducers} from 'redux';
-import {users, user} from './userReducer';
-
+import { users, user} from './userReducer';
+import { combineReducers } from 'redux';
 const rootReducer = combineReducers({
-  users,
-  user
+  user: user,
```

Comment:
Question: did you do on purpose this change? In ES6 it's not required to do that.